### DISCUSSION TITLE
Where to join codespaces beta?

### DISCUSSION THREAD

**Discussion by leorossi2005** [community/community#33385] [Codespaces] [answered]
Where can I join the codespaces beta?

**Comment by zarifpour** [comment, marked as answer]
It is available as a "limited beta release for individual users on GitHub Free and GitHub Pro plans", see the GitHub documentation [here](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/enabling-github-codespaces-for-your-organization) for more info.

**Reply by kbates-verdant** [reply]
That's the only thing it says about the individual users in the documentation. Does that mean I have to wait to be invited, or is the there a waitlist/beta program I can join? I would like to try out Codespaces as an individual before bringing over my organization.

**Reply by otter-computer** [reply]
Yep, unfortunately it's only in a private beta for individual accounts right now. There isn't any news on when it'll open to a public beta or be fully released just yet.
